premium/usage.go

```diff
@@ -240,6 +260,21 @@ func NewUsageClient(meta plugin.Meta, ops ...UsageClientOptions) (UsageClient, e
 	return u, nil
 }
 
+func isAWSMarketplace() bool {
+	return os.Getenv("CQ_AWS_MARKETPLACE_CONTAINER") == "true" || os.Getenv("CQ_AWS_MARKETPLACE_AMI") == "true"
+}
+
+func awsMarketplaceProductCode() string {
+	if os.Getenv("CQ_AWS_MARKETPLACE_CONTAINER") == "true" {
+		return "PLACEHOLDER"
+	}
+	if os.Getenv("CQ_AWS_MARKETPLACE_AMI") == "true" {
+		// TODO: This is a placeholder product code, we need to get the real one from AWS Marketplace
```

Comment:
Maybe read the placeholder from an env variable so we can test this without modifying the code? After we finish testing we can hard code our own value

Reply:
Have already updated it with the real code